sdk/tables/azure-data-tables/tests/test_table_batch.py

```diff
@@ -191,149 +204,191 @@ def test_batch_insert(self, resource_group, location, storage_account, storage_a
             entity.test5 = datetime.utcnow()
 
             batch = self.table.create_batch()
-            batch.create_item(entity)
-            resp = self.table.commit_batch(batch)
+            batch.create_entity(entity)
+            resp = self.table.send_batch(batch)
```

Comment:
We should make some assertions on the response to test the `BatchTransactionResult` object